src/pseudo.cpp

```diff
@@ -791,13 +791,13 @@ uint PseudoSprite::ReadValue(istream& in, width w) {
 			return 0;
 		}
 		int val, err = 0;
-		size_t s = in.tellg();
+		size_t s = (size_t)in.tellg();
 		val = DoCalc(in.ignore(),err);
 		if (err>0)
 			return 0;
 
 		// Replace the original RPN with value
-		size_t e = in.tellg(),
+		size_t e = (size_t)in.tellg(),
```

Comment:
`off_t s = in.tellg();`
and
`off_t e = in.tellg();`
?

Reply:
I tried to use `auto` but then it complains about `e - s` type on lines 801 and 802.

Reply:
In grfid.cpp I used the equivalent of `size_t(e - s)` to do the same (although it's std::size_t there because I was learning ;))

That makes it the correct type without casting.